compiler/testData/diagnostics/tests/typealias/typeAliasAsSuperQualifier.fir.kt

```diff
@@ -22,8 +22,8 @@ class TestSuperForBase : B() {
     override fun foo() {
         super<Base>.foo()
         super<B>.foo()
-        super<<!UNRESOLVED_REFERENCE, UNRESOLVED_REFERENCE!>MyBase<!>>.<!UNRESOLVED_REFERENCE!>foo<!>()
-        <!NOT_A_SUPERTYPE!>super<U><!>.foo()
+        super<<!NOT_A_SUPERTYPE!>MyBase<!>>.foo()
```

Comment:
FE1.0 does accept this because the non-top level `MyBase` is resolved in FE1.0. However, FIR does not honor non-top level type alias, hence `MyBase` is rejected here.